src/Context.php

```diff
@@ -35,8 +38,8 @@ public function __construct($args)
         // algorithm for signing; not necessary for verifying.
         if (isset($args['algorithm'])) {
             $this->algorithmName = $args['algorithm'];
```

Comment:
looks like this class variable is unused

Reply:
Revoled in 0ff0bae479759100fcf0ac42bb6c58324ae6e92d